src/fizzBuzz.js

```diff
@@ -0,0 +1,11 @@
+export default function fizzBuzz(numList){
+  for (var i=0; i < numList.length; i++){
+    if ((numList[i] % 15) === 0){
+      numList.splice([i], 1, 'fizzBuzz')
+    } else if ((numList[i] % 5) === 0){
+      numList.splice([i], 1, 'Buzz')
+    } else if ((numList[i] % 3) === 0){
+      numList.splice([i], 1, 'Fizz')
+    } else null
+  } return numList
+}
```

Comment:
cool thanks for the review Dev! Also here is the Python implementation I was telling you about... 

`def fizzbuzz(n):

    if n % 3 == 0 and n % 5 == 0:
        return 'FizzBuzz'
    elif n % 3 == 0:
        return 'Fizz'
    elif n % 5 == 0:
        return 'Buzz'
    else:
        return str(n)

print "\n".join(fizzbuzz(n) for n in xrange(1, 21))`